src/controller/java/src/chip/devicecontroller/model/Status.java

```diff
@@ -54,4 +54,8 @@ public static Status newInstance(int status, int clusterStatus) {
   public static Status newInstance(int status) {
     return new Status(status, Optional.empty());
   }
+
```

Comment:
could you help the previous newInstance  in this PR, thanks
  public static Status newInstance(int status, int clusterStatus) {
    return new Status(status, Optional.of(clusterStatus));
  }

Reply:
pr created. thanks